### ISSUE TITLE
TypeError when changing disabledDates

### ISSUE THREAD

**Issue by happydude** [btecu/ember-cli-bootstrap-datetimepicker#70]
I'm working on updating a project from ember 3.4 to 3.8. I'm running into a TypeError when changing the dates on the array passed into disabledDates.

```
metal.js:289 Uncaught TypeError: Cannot read property 'apply' of undefined
    at sendEvent (metal.js:289)
    at ObserverSet.flush (metal.js:417)
    at endPropertyChanges (metal.js:855)
    at changeProperties (metal.js:881)
    at setProperties (metal.js:3710)
    at Class.setProperties (observable.js:193)
    at CurlyComponentManager.update (glimmer.js:4435)
    at UpdateComponentOpcode.evaluate (runtime.js:1888)
    at UpdatingVM.execute (runtime.js:4280)
    at RenderResult.rerender (runtime.js:4607)
```

When pausing on exceptions I see an action of updateDisabledDates on InteractiveRenderer.
I saw in addon/components/bs-datetimepicker.js there is an add and remove observer for updateDisabledDates but I cannot find where this is defined.

I was able to duplicate the issue with a test in bs-datetimepicker-test.js https://github.com/btecu/ember-cli-bootstrap-datetimepicker/pull/69

Just changing the array is all that is needed to replicate the error
```
this.set('disabledDates',['2016-01-02','2016-01-03']);
await render(hbs`{{bs-datetimepicker date='2016-01-01' disabledDates=disabledDates openOnFocus=true}}`);
this.set('disabledDates',['2016-01-02','2016-01-03','2016-01-04']);
```

**Comment by btecu** [comment]
Yes, `updateDisabledDates` is missing:
```
  updateDisabledDates() {
    this.picker().disabledDates(this.get('disabledDates'));
  },
```

Can you try add it see if works?

**Comment by happydude** [comment]
Thank you, that seems to have done the trick. I have added it to the pull request

**Comment by btecu** [comment]
Fixed in #69.
